monkey/monkey_island/cc/next_ui/src/app/error.tsx

```diff
@@ -0,0 +1,29 @@
+'use client'; // Error components must be Client Components
```

Comment:
Ideally, comments should explain **why** certain choices were made. Afaik this is already explained in the documentation? "error.js boundaries must be [Client Components](https://nextjs.org/docs/app/building-your-application/rendering/client-components)."